Tabloid/Repositories/CategoryRepository.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using Tabloid.Models;
+using Tabloid.Utils;
+
+namespace Tabloid.Repositories
```

Comment:
Are there DbUtils methods that could be used in this repo to do some of this work? We'll want to use those utils methods consistently across the codebase.

Reply:
resolved @heymonicakay